WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1089,13 +1091,27 @@ SearchHelperToolbarMargin   = margin-top:3px;
 
 SearchResultColor           = #FFFEC4
 
+###################
+#   AUTO COMPLETE SEARCH
+###################
+
+AutoCompleteWaitingContainer = width: 300px; height: 70px; padding: 1.5em; box-sizing: border-box; @Text-hint@ @roundCorners(2px)@; box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5); background-color:@ListItemBgColor-normal@;
+AutoCompleteListViewContainer= @MenuContainer@ background-color:@ListItemBgColor-normal@; width:auto; padding:0px !important; border-width:1px; width: 300px; box-sizing: border-box; 
```

Comment:
Just moving the styles from css to the property value. Can be an enhancement thing but doesn't seem to be part of this ticket.

.ZmAutocompleteListView {
 -	@MenuContainer@
 -	background-color:@ListItemBgColor-normal@;
 -	width:auto;
 -	padding:0px !important;
 -	border-width:1px;
 +	@AutoCompleteListViewContainer@
  }